packages/python/src/components/Variable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Children, code } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js"; // assuming you have this
+
+export interface VariableProps {
+  name: string;
+  value?: Children;
+  type?: Children; // Optional, only for type annotation
+  omitNone?: boolean; // Optional, to omit None assignment
+}
+
+export function Variable(props: VariableProps) {
```

Comment:
AssignmentStatement?

Reply:
The python grammar is AssignmentStatement but I think it's more valuable to have `Declaration` in the name of components which make declarations, and `AssignmentDeclaration` seems weird to me, but I also wouldn't be opposed to that.